dashboard/db/migrate/20170518002933_remove_columns_from_script_levels.rb

```diff
@@ -0,0 +1,37 @@
+class RemoveColumnsFromScriptLevels < ActiveRecord::Migration[5.0]
+  def up
+    ScriptLevel.where(named_level: true).each do |sl|
+      sl.type = ScriptLevel::NAMED_LEVEL
+      sl.save!
+    end
+    ScriptLevel.where(assessment: true).each do |sl|
+      sl.type = ScriptLevel::ASSESSMENT
+      sl.save!
+    end
+    ScriptLevel.where(bonus: true).each do |sl|
+      sl.type = ScriptLevel::BONUS
+      sl.save!
+    end
+
+    remove_column :script_levels, :named_level, :boolean
+    remove_column :script_levels, :assessment, :boolean
+    remove_column :script_levels, :bonus, :boolean
+  end
+
+  def down
+    add_column :script_levels, :named_level, :boolean
+    add_column :script_levels, :assessment, :boolean
+    add_column :script_levels, :bonus, :boolean
+
+    ScriptLevel.all.each do |sl|
+      case sl.type
+      when ScriptLevel::NAMED_LEVEL
+        sl.update!(named_level: true)
+      when ScriptLevel::ASSESSMENT
+        sl.update!(assessment: true)
+      when ScriptLevel::BONUS
+        sl.update!(bonus: true)
+      end
+    end
+  end
+end
```

Comment:
The script cache should mostly insulate us from errors in the time between migrating the DB and upgrading the frontends, so this won't be as bad as coupling the migration and code changes would normally be. Maybe just give DotD a heads up when this PR is going to go to prod?